### DISCUSSION TITLE
Suggestions for UI

### DISCUSSION THREAD

**Discussion by CTBlaze** [monocasual/giada#559] [Ideas]
Ciao Giovanni,
I have some sugestions for the UI   (not quite comperehensively summarized, as I will post them here as they come along, if that's OK.) 

the first two are:

- UI_01 - add the icons for the channel mode in front of the text entries into the drop down list
  (that way, users can relate to the icon and it's textual/semantic meaning already while choosing it, while seeing see options)

- UI_02 - change the display of channel progress "bar" from the small square to a long thin stripe, which is as long as the whole channel box, but only 2-3 pixels in height. On top or at the bottom of the box, what would be better? I reckon on top.  
 (that way, the visual aid of that progress bar is more helpful, to keep timings and keep the overview.)
 --- sub-idea: introduce baby light blue, on top of black, grey and baby light pink. the baby light blue could be used for the progress stripe filling, would be a nice contrast and could act as a "divider" between the channels, in addition

**Comment by gvnnz** [comment]
Ciao @CTBlaze , thanks a lot. Feel free to add here anything you want, we love getting feedback from people ;) I'll start with the first two bullets:

> * UI_01 - add the icons for the channel mode in front of the text entries into the drop down list

This is a very nice suggestion. +1

> * UI_02 - change the display of channel progress "bar" from the small square to a long thin stripe, which is as long as the whole channel box, but only 2-3 pixels in height

I like this one. We actually changed the progress bar layout in the upcoming version 0.20.1 into a vertical bar that spans the entire channel height ([screenshot](https://www.instagram.com/p/CZcyodps6lh/)). Your idea is cool though, I guess we can include it in the future channel redesign.  

>   --- sub-idea: introduce baby light blue, on top of black, grey and baby light pink. the baby light blue could be used for the progress stripe filling, would be a nice contrast and could act as a "divider" between the channels, in addition

I agree, more colors should be added to the "official" theme.
